app/src/test/kotlin/no/nav/arbeidsgiver/notifikasjon/ekstern_varsling/EksternVarslingServiceTests.kt

```diff
@@ -102,7 +102,7 @@ class EksternVarslingServiceTests : DescribeSpec({
             }
 
             it("message received from kafka") {
-                eventually(5.seconds) {
+                eventually(10.seconds) {
```

Comment:
ikke noe vi trenger å gjøre nå, men en ide her kunne vært å ha en config for delays man sender til EksternVarslingService slik at vi kan ha lav delay i test sammenheng. bare en tanke. vi kan la det marinere litt